src/patternfly/components/Popover/docs/code.md

```diff
@@ -1,19 +1,31 @@
 ## Overview
 
-Only if needed write a short description with implementation notes. Design and interaction notes are already written on the design specs, don't repeat information.
-
-For example for buttons: Always add a modifier class to add color to the button. Never use the class `.btn` on its own.
+A popover is used to provide contextual information for another component on click.  The popover itself is made up of two main elements: arrow and content. The content element follows the pattern of the popover box component, with a close icon in the top right corner, as well as a header and body.  One of the directional modifiers (`.pf-m-left`, `.pf-m-top`, etc.) is required on the popover component
 
 ## Accessibility
 
-| Attribute | Applied To | Outcome |
+| Attribute | Applies to | Outcome |
 | -- | -- | -- |
-| `role` or `aria` | `pf-c-popover` |  accessibility notes. |
+| `role="dialog"` | `.pf-c-popover` | Identifies the element that serves as the popover container. **Required**|
+| `aria-labeledby="[id value of .pf-c-popover__header-title]"` | `.pf-c-popover` | Gives the popover an accessible name by referring to the element that provides the dialog title. **Required when .pf-c-popover__header-title is present** |
+| `aria-label="[title of popover]"` | `.pf-c-popover` | Gives the popover an accessible name. **Required when .pf-c-popover__header-title is _not_ present** |
+| `aria-describedby="[id value of applicable content]"` | `.pf-c-popover` | Gives the popover an accessible description by referring to the popover content that describes the primary message or purpose of the dialog. Not used if there is no static text that describes the popover. |
+| `aria-popover="true"` | `.pf-c-popover` | Tells assistive technologies that the windows underneath the current popover are not available for interaction. **Required**|
+| `aria-label="Close Dialog"` | `.pf-c-popover__close .pf-c-button` | Provides an accessible name for the close button as it uses an icon instead of text. **Required**|
```

Comment:
@mcarrano Did you have a text string you wanted to use for the Close buttons on popovers? This looks like it was copied over from the Modal example, but I'm assuming we'd want something different for popovers.

Reply:
@jgiardino Do we even need the label to reference the object? What if we just say "Close?"

Reply:
That works for me. Would you want the same label for the ModalBox example? If so, we should open an issue to update that one, too.

Reply:
Yes, I think so @jgiardino . We shouldn't assume users will know what we mean when we use words like "dialog" or "popover." Aren't popovers some kind of puff pastry? :)

Reply:
But that would go nicely with our food theme: kebab, hamburger, toast... 

I opened an issue to capture the text string changes for both this and the modal so that we can track that change separately from this PR: https://github.com/patternfly/patternfly-next/issues/525